modin/pandas/series_utils.py

```diff
@@ -129,17 +135,24 @@ def _default_to_pandas(self, op, *args, **kwargs):
 
 @_inherit_docstrings(pandas.core.strings.StringMethods)
 class StringMethods(ClassLogger):
-    def __init__(self, series):
+    def __init__(self, data):
         # Check if dtypes is objects
 
-        self._series = series
-        self._query_compiler = series._query_compiler
+        self._series = data
+        self._query_compiler = data._query_compiler
+
+    @pandas.util.cache_readonly
+    def _Series(self):  # noqa: GL08
```

Comment:
@dchigarev I changed my approach a bit. However `__constructor__` is not used here so that the return value of the property can be used both as a type and as a constructor.